src/App.tsx

```diff
@@ -16,34 +16,95 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [activeSort, setActiveSort] = useState<string>('all');
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  enum SortStatus {
+    All = 'all',
+    Alphabetical = 'alphabetical',
+    Length = 'length',
+  }
+
+  function getSort(): string[] {
```

Comment:
Move this function outside of the component

```suggestion
  function getSort(Goods: string[], activeSort: SortStatus, isReserved: boolean): string[] {
```